client/components/navbar/navbar.service.js

```diff
@@ -1,226 +1,481 @@
-(() => {
-    "use strict";
-
-    class NavbarService {
-        constructor ($translate, atInternet, OvhApiMe, OtrsPopupService, ssoAuthentication, TranslateService,
-                     LANGUAGES, TARGET, MANAGER_URLS, REDIRECT_URLS, URLS) {
-            this.$translate = $translate;
-            this.atInternet = atInternet;
-            this.User = OvhApiMe;
-            this.TranslateService = TranslateService;
-            this.OtrsPopupService = OtrsPopupService;
-            this.ssoAuthentication = ssoAuthentication;
-            this.LANGUAGES = LANGUAGES;
-            this.MANAGER_URLS = MANAGER_URLS;
-            this.REDIRECT_URLS = REDIRECT_URLS;
-            this.URLS = URLS;
-            this.TARGET = TARGET;
-            this.menuItems = {};
+class ManagerNavbarService {
```

Comment:
We should improve this file, there is a lot of duplication. There are no quick wins here and I can't blame you, because there are a lot of duplication in the sidebar in the first place.